idunn/blocks/wikipedia.py

```diff
@@ -108,6 +113,46 @@ def wrapper(*args, **kwargs):
 
         return wrapper
 
+class WikipediaCache:
+    _cache = None
```

Comment:
we need to set the expire timeout value somewhere, and make it configurable via settings.
Persist the wikipedia values for 24 hours is probably a good default.

Reply:
done.